.github/workflows/template_autodev.yml

```
@@ -57,11 +61,22 @@ jobs:
     # check to not trigger if dependabot did something or PR was closed or label is not the configured dev label
     if: github.actor != 'dependabot[bot]' && (github.event_name == 'push' || github.event.label.name == inputs.label || github.event.action == 'closed')
 
+    env:
+      HAVING_APP_SECRETS: ${{ secrets.app_id != '' && secrets.private_key != '' }}
```

```suggestion
      USING_APP_CREDENTIALS: ${{ secrets.app_id != '' && secrets.private_key != '' }}
```

I would rephrase this but in general the PR looks good to me